lib/perl/Genome/Command/DelegatesToResult.pm

```diff
@@ -97,7 +105,7 @@ sub __errors__ {
         }
     }
 
-    unless(defined $self->requestor) {
+    unless(defined $self->requestor or defined $self->result_users) {
```

Comment:
I was reluctant to go too far down the road of duplicating the validation that already occurs inside `Genome::SoftwareResult::User`.  If the caller gave us a `result_users` value, it seems like they're at least trying to comply (and we know they didn't use the CLI), so it might be less important to catch it early here to display an error.

Do you think it'd be better to be more thorough here?